widgets/pulseaudio.lua

```diff
@@ -33,44 +33,49 @@ local function factory(args)
     --  '/device\\.string/ p' prints the line containing 'device.string:'.
     
     local settings    = args.settings or function()
+            widgettext  = volume_now.left .. "%"
+            tooltiptext = volume_now.left .. "% (" .. devicetype .. " " .. volume_now.index .. " \"" .. volume_now.device .. "\")"
             if volume_now.muted == "yes" then
-                widget:set_text("Mute")
-            else
-                widget:set_text(volume_now.left .. "%")
+                widgettext  = "Mute"
+                tooltiptext = "Mute/" .. tooltiptext
             end
+            widget:set_text(widgettext)
+            tooltip:set_text(tooltiptext)
         end
-        
+    
     pulseaudio.widget = wibox.widget.textbox()
-
+    pulseaudio.tooltip = awful.tooltip({ objects = { pulseaudio.widget } })
```

Comment:
This is again just a personal settings: you can add it to `pulseaudio.widget` from `rc.lua`.

And you can write this too in the wiki entry, in a dedicated section.